kolibri/utils/options.py

```diff
@@ -136,6 +136,13 @@ def calculate_thread_pool():
                 "KOLIBRI_CENTRAL_CONTENT_BASE_URL",
                 "CENTRAL_CONTENT_DOWNLOAD_BASE_URL",
             ),
+        },
+        "DATA_PORTAL_SYNCING_BASE_URL": {
```

Comment:
What is the use case for making this customizable? We want to allow syncing against arbitrary Kolibri devices, but there is only one data portal.

Reply:
If it is just for testing purposes, I'd rather we just change it adhoc.

Reply:
I am imagining the case where we set up custom data portal servers for specific partners. 
But also, it is helpful for testing locally.